test/test_linearizer.py

```diff
@@ -195,6 +197,8 @@ def test_mid_dim_multireduce(self):
     wanna_output = (x.numpy()-x.numpy().sum(axis=1, keepdims=True)).sum(axis=1).reshape(27,1,1,5)
     lins = helper_linearizer_ast(sink, [x], wanna_output=[wanna_output], opts=opts)
     for l in lins:
+      if Device.DEFAULT not in {"METAL"}:
```

Comment:
why is METAL=1 different? is this the global_max thing?

Reply:
it was because I had reuse disabled if `opts` had `tensor_cores`. I'll check if we still need it

Reply:
seems to work, pushing a diff

Reply:
it's not the job of the reviewer to tell you these, when you open a diff for review you must read it 3 times before and question every line yourself.
I can't really trust this diff to be complete enough to merge it.